src/third_party_lib/wirehair/CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ set(GEN_TABLES
         gf256.h
         )
 
-if(NOT ANDROID)
+if(NOT ANDROID AND NOT USE_WASM)
     set(ARCH_NATIVE "-march=native")
```

Comment:
The reason for pause I have is that low-power devices (i.e. android phones) are where we'd really benefit from the optimization -- and we're not doing it there anyway. So it might make sense to toss this altogether.